### ISSUE TITLE
Add Support for Bulk Tracking

### ISSUE THREAD

**Issue by MarcoWel** [karrioapi/karrio-dashboard#262]
**Is your feature request related to a problem? Please describe.**
For tracking external shipments with karrio Dashboard, a tracker needs to be created one by one and the carrier needs to be chosen. This is tedious and time consuming.

**Describe the solution you'd like**
There should be a bulk tracking feature in karrio Dashboard where I can copy&pase tracking numbers into a text box (one by line), and karrio is auto-detecting the carrier and adding trackers automatically.

**Describe alternatives you've considered**
Other services like 17track have this feature:
https://www.17track.net

**Additional context**
--

**Comment by MarcoWel** [comment]
Carrier detection can be done mostly by regex - the regex can be pre-defined in the carrier settings.
```
UPS:         (1Z[A-Z0-9]{6}[0-9]{10})
DPDHL:       (0034[0-9]{16}|CO[0-9]{9}[A-Z]{2}|)
DHL Express: ([0-9]{10}|JD[0-9]{18})
```
DPDHL and DHL Express can/should both be tracked by the DHL Unified Tracking API.

**Comment by MarcoWel** [comment]
I see there is a similar milestone on the roadmap already:
https://github.com/orgs/karrioapi/projects/12/views/1?pane=issue&itemId=10124218

**Comment by danh91** [comment]
That's right. It's on the roadmap. Hopefully, we will tackle it very soon

**Comment by MarcoWel** [comment]
More reading on this topic:
- https://github.com/jkeen/tracking_number_data
- https://stackoverflow.com/questions/619977/regular-expression-patterns-for-tracking-numbers
